plugins/scaffolder-backend/package.json

```diff
@@ -87,7 +87,7 @@
     "node-fetch": "^2.6.7",
     "nunjucks": "^3.2.3",
     "octokit": "^2.0.0",
-    "octokit-plugin-create-pull-request": "^3.10.0",
+    "octokit-plugin-create-pull-request": "4.1.1",
```

Comment:
it looks like there are new versions of this package available now so I believe we can bump this further and add the `^`

See https://github.com/gr2m/octokit-plugin-create-pull-request/releases